udata/api/__init__.py

```diff
@@ -106,6 +106,9 @@ def wrapper(*args, **kwargs):
             if current_user.deleted:
                 self.abort(401)
 
+            if current_user.password_rotation_demand:
```

Comment:
do we rely on user/password at the API level? Is it not only API key related?

Reply:
Yes i might have been a bit zealous.